static_frame/core/util.py

```diff
@@ -660,6 +660,35 @@ class IterNodeType(Enum):
     ITEMS = 2
 
 
+class SortStatus(Enum):
+    ASC = 1
+    DESC = 2
+    NO = 3
```

Comment:
Small quibble: `NO` feels like a different tense than `ASC`  and `DESC`; maybe "NA" or "UNSORTED" or something else...

Reply:
I wondered about this haha. I like `NA` best!